skimage/data/tests/test_data.py

```diff
@@ -151,6 +154,3 @@ def test_cells_3d():
     path = fetch('data/cells.tif')
```

Comment:
Looks like this should be updated to reflect the changes in this PR.

Reply:
thanks!